### DISCUSSION TITLE
Xbox series x support!

### DISCUSSION THREAD

**Discussion by massy-h** [albar965/littlenavmap#885] [Ideas]
Hello Sir,

First I wanted to thank you for such an effort to create littlenavmap, such a great application!

secondly, it would be really great if you could add xbox series x support to your code, eg running the littlenavmap on a pc to remotely connect to MSFS 2020 running on xbox x. 

Regards,
Massy

![merlin_179317002_bdab34db-887e-4d98-9d35-989134dda4af-articleLarge](https://user-images.githubusercontent.com/35938759/168428104-f1e64c33-88bc-40ab-bb86-22740b760e8c.jpg)

**Comment by albar965** [comment]
Sorry, this is not doable for now. There is no SimConnect for XBox MSFS available: https://forums.flightsimulator.com/t/simconnect-support-for-xbox-editions/425862

**Reply by massy-h** [reply]
Thanks for the info, although it's quite disappointing! 🙂
